runtime/CSharp/Antlr4BuildTasks/Antlr4ClassGenerationTaskInternal.cs

```diff
@@ -301,7 +301,7 @@ public bool Execute()
                     string framework = TargetFrameworkVersion;
                     if (string.IsNullOrEmpty(framework))
                         framework = "v2.0";
-                    if (framework.StartsWith("v4.5."))
+                    else if (framework.StartsWith("v4.5.") || framework.Equals("v4.6"))
```

Comment:
I like the `Version` class approach better. But is it safe to target all future .NET versions? Or does it make sense to recognize 4.5.x and 4.6.x only?